client/state/themes/themes-ui/selectors.js

```diff
@@ -26,8 +26,3 @@ export function getBackPath( state ) {
 export function isThemesBannerVisible( state ) {
 	return state.themes.themesUI.themesBannerVisible;
 }
-
-//  Returns true if the theme showcase banner is currently visible
-export function isThemesUpworkBannerVisible( state ) {
```

Comment:
The `isThemesUpworkBannerVisible` function is still referenced in `client/state/themes/themes-ui/test/selectors.js`.

Reply:
removed from tests